source/includes/_build.md

```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
 
 ## Linux or macOS
 
-For Iroha, a provisioning of Postgres and Redis are required. You may launch them on your local machine, or use docker containers, as provided.
-
 > Launching Docker and Postgres in Docker
 
 ``` shell
-docker run --name some-redis -p 6379:6379 -d redis:3.2.8
-docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres:9.5
+docker run --name some-redis \
+-p 6379:6379 -d redis:3.2.8
+docker run --name some-postgres \
+-e POSTGRES_USER=iroha \
+-e POSTGRES_PASSWORD=helloworld \
+-p 5432:5432 -d postgres:9.5
 ```
 
-> Linux
+To launch Iroha daemon, running postgres and redis services are required. You may launch them on your local machine, or use docker containers, as provided on the right side.
 
-``` shell
-apt-get -y --no-install-recommends install \
-        build-essential python-software-properties \
-        automake libtool \
-        libssl-dev zlib1g-dev libboost-all-dev \
-        libc6-dbg golang \
-        git ssh tar gzip ca-certificates \
-        python3 python3-pip python3-setuptools \
-        lcov \
-        wget curl cmake file unzip gdb \
-        iputils-ping vim ccache \
-        gcovr vera++ cppcheck doxygen \
-        graphviz graphviz-dev; \
-    apt-get -y clean
-git clone https://github.com/hyperledger/iroha -b develop
-cd iroha
-cmake -H. -Bbuild
-cmake --build build -- -j4
-```
+### Linux (debian-based)
 
-> macOS
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/39557aa444c3317aeffbdedd9f0f51e2.js"></script>
```

Comment:
Looks like tag <script> didn't work (

Reply:
@neewy will it work in slate?

Reply:
<img width="737" alt="screen shot 2017-12-16 at 16 50 26" src="https://user-images.githubusercontent.com/11841667/34071058-3cc7098c-e281-11e7-9631-b94543042600.png">